### DISCUSSION TITLE
How to Disable Auto-Starting Minecraft Server in the Container

### DISCUSSION THREAD

**Discussion by dam2452** [itzg/docker-minecraft-server#3001] [General]
Hi,

I am using this Minecraft server container, and I need to modify the behavior so that the Minecraft server does not start automatically when the container is launched. Could you please provide guidance on how to disable the auto-start of the Minecraft server within the container? Additionally, could you let me know where the script that starts the Minecraft server is located?

Thank you for your assistance!

**Comment by itzg** [comment]
Like any container image, the entrypoint is declared here

https://github.com/itzg/docker-minecraft-server/blob/a71c5384b420c75f2dc4412375a418535b504c2e/Dockerfile#L76

You can override that in your container config. 

...however, I'm curious to know why you want to even consider this. It seems like a red flag that you're doing something wrong.
